### PR TITLE
comics and auto mods.md generation

### PR DESCRIPTION
**Pull request by asdqwex**

### CODE REVIEW

File: src/modules/comics.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+var request = require('request')
+
+module.exports = {
+  names: ['comic'],
+  onMessage: function (bot, user, userID, channelID, message, rawEvent) {
+    var term = message.split(' ').splice(1).join(' ')
+    if (term === 'c&h') {
```
Comment (Kingles) [inline src/modules/comics.js:9]: Could be useful to match others as well. Help lists explosm but you can't use 'explosm' as a command

**Comment by Kingles** [comment]
In the future can you make separate commits for each feature/enhancement/bugfix? makes it easier to roll back if there are issues.

**Comment by asdqwex** [comment]
yessir! o7

**Comment by asdqwex** [comment]
Failing this

**Comment by asdqwex** [comment]
Manually merged since I made a mess trying to cherry pick my commits back in. Moving on.

**Pull request merged**